azure/packages/external-controller/package.json

```diff
@@ -47,12 +47,14 @@
 		"@fluidframework/azure-client": "workspace:~",
 		"@fluidframework/common-definitions": "^0.20.1",
 		"@fluidframework/routerlicious-driver": "workspace:~",
+		"@fluidframework/telemetry-utils": "workspace:~",
 		"assert": "^2.0.0",
 		"axios": "^0.26.0",
 		"events": "^3.1.0",
 		"fluid-framework": "workspace:~"
 	},
 	"devDependencies": {
+		"@fluid-experimental/devtools": "workspace:~",
```

Comment:
Has to be a dev dependency to avoid violating our policy checks (same reason `@fluidframework/test-client-utils` is under devDependencies).